packages/utilities/moment-timezone-include/src/writeToNestedFile/index.js

```diff
@@ -0,0 +1,19 @@
+import fs from 'fs';
+import mkdirp from 'mkdirp';
+import path from 'path';
+
+const getDirOfPath = filePath =>
+  filePath.substring(0, filePath.lastIndexOf('/') + 1);
+
+const writeToNestedFile = (filePath, contents) => {
+  const fullFilePath = path.join(__dirname, filePath);
+  const folder = getDirOfPath(fullFilePath);
```

Comment:
You can use `path.dirname(fullFilePath)` here instead of your custom function. https://nodejs.org/api/path.html#path_path_dirname_path

Reply:
Oooh niceee, thanks